web_app/tests/test_user.py

```diff
@@ -81,33 +95,38 @@ async def test_check_user(async_client, wallet_id: str):
             "0x27994c503bd8c32525fbdaf9d398bdd4e86757988c64581b055a06c5955ea49",
             "0x698b63df00be56ba39447c9b9ca576ffd0edba0526d98b3e8e4a902ffcf12f0",
         ),
+        ("invalid_wallet_id", None),
+        (123_456_789, None),
+        (3.14, None),
+        ({}, None),
     ],
 )
 async def test_change_user_contract(
-    async_client, wallet_id: str, contract_address: str
+    client: client,
+    mock_user_db_connector: MagicMock,
+    wallet_id: str,
+    contract_address: str,
 ):
     """
-    Test get_user_contract endpoint,
-    where wallet_id = "0x27994c503bd8c32525fbdaf9d398bdd4e86757988c64581b055a06c5955ea49" or ""
-    and contract_address = "0x698b63df00be56ba39447c9b9ca576ffd0edba0526d98b3e8e4a902ffcf12f0" or ""
-    :param async_client: httpx.AsyncClient
+    Test get_user_contract endpoint
+    :param wallet_id: "" or "0x27994c503bd8c32525fbdaf9d398bdd4e86757988c64581b055a06c5955ea49"
+    :param contract_address: "" or "0x698b63df00be56ba39447c9b9ca576ffd0edba0526d98b3e8e4a902ffcf12f0"
     :return: None
     """
     data = UpdateUserContractRequest(
-        wallet_id=wallet_id,
-        contract_address=contract_address,
+        wallet_id=str(wallet_id),
```

Comment:
Why do you stringify a string?

Reply:
because of in parameters which is in decorator we have negative scenarios which will raise error because they are not a string, but list, dict, float etc.